dallinger/data.py

```diff
@@ -161,7 +161,11 @@ def export(id, local=False, scrub_pii=False):
 
     print("Preparing to export the data...")
 
-    copy_heroku_to_local(id)
+    if local:
+        db_name = 'dallinger'
```

Comment:
I don't think the config's `database_url` ever overrides the default in debug mode. However, the `DATABASE_URL` environment variable can override it. I'll switch this to use `dallinger.db.db_url` instead.